server/data/hmppsAuthClient.test.ts

```diff
@@ -0,0 +1,103 @@
+import nock from 'nock'
+import redis from 'redis'
+
+import config from '../config'
+import HmppsAuthClient from './hmppsAuthClient'
+
+const username = 'Bob'
+const token = { access_token: 'token-1', expires_in: 300 }
+
+jest.mock('redis', () => ({
```

Comment:
Think extracting redis client into separate component would make testing this much easier - add a bit of separation of concerns